cml-tdengine/src/core/register.rs

```diff
@@ -266,6 +257,15 @@ mod tests {
             .await?
             .unwrap_or(0);
         assert_eq!(4, records);
+        // check tag count
+        let tag_records = taos
+            .query_one(format!(
+                "SELECT COUNT(*) FROM INFORMATION_SCHEMA.INS_TAGS WHERE db_name = 'training_data' and table_name = '{}'",
+                another_metadata_check.batch()
+            ))
+            .await?
+            .unwrap_or(0);
+        assert_eq!(3, tag_records);
```

Comment:
Can you explain the goal of this check?

Reply:
Check whether the count of labels is inserted correctly by using the code below.
[register.rs#L52-L56](https://github.com/bioinformatist/cml/blob/0b7886f24934d3b6c574d21aadb10c024d98cbf3/cml-tdengine/src/core/register.rs#L52-L56)